comictaggerlib/taggerwindow.py

```diff
@@ -1408,28 +1411,39 @@ def adjust_source_combo(self) -> None:
         self.cbx_sources.setCurrentIndex(self.cbx_sources.findData(self.config[0].Sources__source))
 
     def adjust_load_style_combo(self) -> None:
-        # select the current style
-        self.cbLoadDataStyle.setCurrentIndex(self.cbLoadDataStyle.findData(self.load_data_style))
+        # select the enabled styles
+        unchecked = set(metadata_styles.keys()) - set(self.load_data_styles)
+        for i, style in enumerate(self.load_data_styles):
+            item_idx = self.cbLoadDataStyle.findData(style)
+            self.cbLoadDataStyle.setItemChecked(item_idx, True)
+            # Order matters, move items to list order
+            self.cbLoadDataStyle.moveItem(item_idx, row=i)
+        for style in unchecked:
+            self.cbLoadDataStyle.setItemChecked(self.cbLoadDataStyle.findData(style), False)
 
     def adjust_save_style_combo(self) -> None:
         # select the current style
         unchecked = set(metadata_styles.keys()) - set(self.save_data_styles)
         for style in self.save_data_styles:
-            self.cbSaveDataStyle.setItemChecked(self.cbLoadDataStyle.findData(style), True)
+            self.cbSaveDataStyle.setItemChecked(
+                self.cbSaveDataStyle.findData(style), True
+            )  # Why were these read style?
```

Comment:
I think I went back in the history and it was always `self.cbLoadDataStyle.findData(style)`. It's going to be the same `cr` etc. but seemed odd. Possibly a copy/paste?